src/game/game_player_record.py

```diff
@@ -7,7 +7,14 @@ class GamePlayerRecord(polymodel.PolyModel):
     player_key = ndb.KeyProperty(kind="User", required=True)
     player_placement = ndb.IntegerProperty(required=True,
                                            choices=[1, 2, 3, 4, 5])
-    rank_points_earned = ndb.IntegerProperty()
+    rank_points_earned = ndb.FloatProperty(default=0.0)
```

Comment:
Why use a float? I think that it might be worth leaving this as an integer and rounding. 
I just think it might be funny to have a skill rank of 47.2
Maybe round this float when we display it?